virt/managing_vms/virt-delete-vms.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can delete a virtual machine from the web console or by using the `oc` command-line interface.
+You can delete a virtual machine by using the web console or the `oc` command line interface.
```

Comment:
command-line should be hyphenated when it's an adjective in front of something like "interface".

But again, you could use {oc-first} if you want